### PR TITLE
Enhancement registration form

### LINKED ISSUE #652: Enhancement of registration form
[Link to the page](https://girlscriptboilerplate.netlify.app/registrations)

1. Add header and footer
2. Add Vectors 
**Any more enhancements will be welcome.**

### PR DESCRIPTION
**Pull request by Adyasha8105**
- I have added footer and navbar ( using another js script).
- I have removed the wave section.
- I have removed the bug present on the registration page.
- I have made the **page responsive.**

@anushbhatia Please check it.

### CODE REVIEW

**Comment by aashimgarg** [comment]
@anushbhatia i want to ask that footer and header are dynamic why needed to add it again.

File: registrations.html [hunk 1]
```diff
@@ -1,10 +1,82 @@
...
+		<script src="https://code.jquery.com/jquery-3.5.0.min.js"
+  		integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
+  		crossorigin="anonymous"></script>
+	<!-- Bootstrap -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+	<!-- Normalize CSS and AOS -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+	<link rel='stylesheet' href='https://unpkg.com/aos@2.3.0/dist/aos.css'>
+	<script src='https://unpkg.com/aos@2.3.0/dist/aos.js'></script>
```
Comment (aashimgarg) [inline registrations.html:75]: @anushbhatia why these libraries are added?

**Comment by anushbhatia** [comment]
You need to atleast call the app. Js file to it.

**Comment by Adyasha8105** [comment]
> You need to atleast call the app. Js file to it.

Sir, actually I have removed the wave part that's why I have to rewrite the code again.

**Comment by Adyasha8105** [comment]
Here, also I made the page responsive.

File: registrations.html [hunk 2]
```diff
@@ -1,10 +1,82 @@
...
+		href="https://girlscriptboilerplate.netlify.app/registration.html" />
+
+	<link rel="next" title="GirlScript Foundation-Chennai Chapter | Registration"
+		href="https://girlscriptboilerplate.netlify.app/registration.html" />
+
+	<link rel="search" href="https://girlscriptboilerplate.netlify.app/registration.html"
+		type="application/opensearchdescription+xml" title="GirlScript Foundation-Chennai Chapter | Registration" />
+
+	<!-- Google Font Api -->
+	<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
```
Comment (aashimgarg) [inline registrations.html:60]: is Lato being used here?

File: registrations.html (hunk 1 above)
Comment (aashimgarg) [reply registrations.html:75]: ??

**Comment by aashimgarg** [comment]
> > You need to atleast call the app. Js file to it.
> 
> Sir, actually I have removed the wave part that's why I have to rewrite the code again.

try to find a better alternative for this rather rewriting the whole code.

**Comment by Adyasha8105** [comment]
> > > You need to atleast call the app. Js file to it.
> > 
> > 
> > Sir, actually I have removed the wave part that's why I have to rewrite the code again.
> 
> you can try to find a better alternative for this rather rewriting the whole code.

@aashimgarg And you can try to give your valuable suggestions in slack in the form of new PR rather than sneaking into someone others PR :)

**Comment by aashimgarg** [comment]
> > > > You need to atleast call the app. Js file to it.
> > > 
> > > 
> > > Sir, actually I have removed the wave part that's why I have to rewrite the code again.
> > 
> > 
> > you can try to find a better alternative for this rather rewriting the whole code.
> 
> @aashimgarg And you can try to give your valuable suggestions in slack in the form of new PR rather than sneaking into someone others PR :)

Its not about any of the individuals its about the org.The best should come out in which ever way we can. And most importantly its an open source contribution if you dont get that not my issue.

**Comment by Adyasha8105** [comment]
> > > > > You need to atleast call the app. Js file to it.
> > > > 
> > > > 
> > > > Sir, actually I have removed the wave part that's why I have to rewrite the code again.
> > > 
> > > 
> > > you can try to find a better alternative for this rather rewriting the whole code.
> > 
> > 
> > @aashimgarg And you can try to give your valuable suggestions in slack in the form of new PR rather than sneaking into someone others PR :)
> 
> Its not about any of the individuals its about the org.The best should come out in which ever way we can. And most importantly its an open source contribution if you dont get that not my issue.

@aashimgarg Everyone's definition of best is different, I did what I felt best.. and if you have some more better suggestions you must create your personal issue or contact the mentors directly instead of always commenting on others PR. 
Good Luck.

**Pull request merged**